pkg/pmk/common.go

```
@@ -14,7 +15,7 @@ func setupNode(hostOS string, exec cmdexec.Executor) (err error) {
 		return err
 	}
 	if err := swapOffFstab(exec, "/etc/fstab"); err != nil {
-		return err
+		return fmt.Errorf("Unable to edit file /etc/fstab")
```

This is going to mask the actual errors and make it difficult for us to debug the failures.
Could you also add a debug log with actual err.
Same comment for other instances as well.

---

@anupbarve The debug log for this is printed here https://github.com/platform9/pf9ctl/blob/622631834f501d061efcf78e14b59f67c1fc4f04/pkg/cmdexec/executor.go#L57